docs/content/docs/custom-resource/reference.md

```diff
@@ -225,6 +225,16 @@ This page serves as a full reference for FlinkDeployment custom resource definit
 | error | java.lang.String | Error information about the FlinkDeployment/FlinkSessionJob. |
 | reconciliationStatus | org.apache.flink.kubernetes.operator.crd.status.FlinkSessionJobReconciliationStatus | Status of the last reconcile operation. |
 
+### JobHealthInfo
+**Class**: org.apache.flink.kubernetes.operator.crd.status.JobHealthInfo
```

Comment:
We should move this class to another package, so that the reference generator does not pick it up. This object is not directly part of the status (only in a serialized form)

Reply:
Moved.